### ISSUE TITLE
A unit converter to easily translate different types of units to each other

### ISSUE THREAD

**Issue by Jay-Madden** [ClemsonCPSC-Discord/ClemBot#99]
This is a great feature as it can be as simple or as complex as the implementer would like. and there are many ways to tackle this. This would be a great first issue for someone to get their feet wet with discord bots

**Comment by aferenc** [comment]
Since nobody's commented on this one, I might try taking it on as my first try with making a cog and service. Though since I'm still fairly new to Python, I can't say whether or not I'd be successful in implementing something anytime soon.

**Comment by Jay-Madden** [comment]
Ohh perfect, i would def suggest looking into a library or api to do this for you, up too you which path you choose. feel free to ping me with any questions

**Comment by aferenc** [comment]
Yeah, I'm sure Python has some sort of library out there that can handle the actual unit conversion formulas. Cause writing out each one by hand as individual functions sounds like a nightmare. I guess my job would just be coming up with a way to take user input, validate it, and feed it to the appropriate conversion function and display the output to the user.
